wolfcrypt/src/asn.c

```diff
@@ -30431,9 +30431,9 @@ int wc_SetCustomExtension(Cert *cert, int critical, const char *oid,
 
     ext = &cert->customCertExt[cert->customCertExtCount];
 
-    ext->oid = oid;
+    ext->oid = (char*)oid;
```

Comment:
I'm still waiting for upstream's feedback. Casting const to non-const is dangerous.